cmd/edgectl/aes_install_errors.go

```diff
@@ -212,7 +146,7 @@ func (i *Installer) WaitForAESError(err error) Result {
 
 	return Result{
 		Report:       "fail_wait_aes",
-		ShortMessage: "An error occurred while applying Kubernetes manifests",
+		ShortMessage: "An error occurred while applying AES Chart",
 		Message:      fmt.Sprintf("Visit %v for a more detailed explanation and suggestions on how to continue installing Ambassador Edge Stack.", url),
 		URL:          url,
```

Comment:
This needs to be a new help page (with a new URL).

Reply:
Ok, fixed.